motr/setup.c

```diff
@@ -2231,6 +2240,16 @@ static int _args_parse(struct m0_motr *cctx, int argc, char **argv)
 				{
 					rctx->rc_be_seg_preallocate = true;
 				})),
+			M0_NUMBERARG('O', "balloc no of groups per ad stob",
+				LAMBDA(void, (int64_t nr)
+				{
+					rctx->rc_balloc_group_nr = nr;
+				})),
+			M0_NUMBERARG('X', "BE balloc no of index in balloc db",
+				LAMBDA(void, (int64_t nr)
+				{
+					rctx->rc_balloc_index_nr = nr;
+				})),
```

Comment:
These parameters are only valid for mkfs, right?
For a m0d, after mkfs is done, these parameters should already be configured and cannot be changed. Is this right?

Reply:
Yes this is correct.  Mkfs use this command line parameter. m0d always read from seg0. There are two methods
 balloc_cfg_get_from_config() from configuration parameter used by mkfs.
  balloc_cfg_get_from_adom() from m0_stob_ad_domain used by m0d.

Reply:
OK